apps/chat/src/components/Marketplace/SearchHeader.tsx

```diff
@@ -31,6 +34,10 @@ export const SearchHeader = ({
 
   const dispatch = useAppDispatch();
 
+  const enabledFeatures = useAppSelector(
+    SettingsSelectors.selectEnabledFeatures,
+  );
```

Comment:
use `isFeatureEnabled` instead
```suggestion
  const isCustomApplicationsEnabled = useAppSelector((state) =>
    SettingsSelectors.isFeatureEnabled(state, Feature.CustomApplications),
  );
```
```suggestion
  const enabledFeatures = useAppSelector(
    SettingsSelectors.selectEnabledFeatures,
  );
```